src/Neuron/Zettelkasten/View.hs

```diff
@@ -39,6 +41,9 @@ import qualified Rib.Parser.MMark as MMark
 import Text.MMark (useExtensions)
 import Text.Pandoc.Highlighting (styleToCss, tango)
 
+searchScript :: Text
+searchScript = $(embedStringFile "./src-script/neuron-web-search/search.js")
```

Comment:
This file should be placed in `./src-js/search.js`. 

`src-script` is for shell scripts.